hazelcast/include/hazelcast/client/ClientConfig.h

```diff
@@ -296,7 +296,7 @@ namespace hazelcast {
             *
             *  @return serializationConfig
             */
-            SerializationConfig const &getSerializationConfig() const;
+            SerializationConfig &getSerializationConfig();
```

Comment:
if it is const, then we can not do getSerializationConfig().registerSerializer(). The only way then to registerSerializer is the ClientConfig.setSerializationConfig(). This was the motivation behind it. I was to use it in a test but I later did not use it but I thought we better fix this as well.